src/test/resources/reference.conf

```diff
@@ -91,6 +91,12 @@ swagger {
   realm = "broad-dsde-test"
 }
 
+monitor {
+  pollPeriod = 1 second
```

Comment:
This was previously 100 millis, hardcoded in the test. I changed it to 1 second and moved it to test config.